sources/packages/backend/apps/queue-consumers/src/processors/schedulers/atbc-respone-integration/atbc-response-integration.scheduler.ts

```diff
@@ -0,0 +1,40 @@
+import { InjectQueue, Process, Processor } from "@nestjs/bull";
+import { Job, Queue } from "bull";
+import { BaseScheduler } from "../base-scheduler";
+import { QueueNames } from "@sims/utilities";
+import { QueueService } from "@sims/services/queue";
+import { ProcessPDRequestQueueOutDTO } from "./models/atbc-response-integration.dto";
+import { ATBCIntegrationProcessingService } from "@sims/integrations/atbc-integration";
+
+/**
+ * Process all the applied PD requests to verify the status with ATBC.
+ */
+@Processor(QueueNames.ATBCResponseIntegration)
+export class ATBCResponseIntegrationScheduler extends BaseScheduler<void> {
+  constructor(
+    @InjectQueue(QueueNames.ATBCResponseIntegration)
+    schedulerQueue: Queue<void>,
+    queueService: QueueService,
+    private readonly atbcIntegrationProcessingService: ATBCIntegrationProcessingService,
+  ) {
+    super(schedulerQueue, queueService);
+  }
+
+  /**
+   * Process all the applied PD requests to verify the status with ATBC
+   * and update the database with processed records.
+   * @param job ATBC response integration job.
+   * @returns processing result.
+   */
+  @Process()
+  async processPendingPDRequests(
+    job: Job<void>,
+  ): Promise<ProcessPDRequestQueueOutDTO> {
+    await job.log("Processing PD status for students.");
```

Comment:
mm... since signori.. merged the new summary logger. are we using it in this PR or are we planning to have a different PR for it?

Reply:
Actually existing PR is stretched beyond the story scope and didn't want add more complexity to the PR. Having said that I wouldn't like leaving a code which is evidently going to be refactored and the solution is straight forward. Will change it.

Reply:
The initial idea was to use the `QueueProcessSummary` at `queue-consumers` level only but we can change it to be a more generic `ProcessSummary` to replace the ones used internally in the integrations. We need also just abstract the loggers provided as parameters.
I can change it in my upcoming PR if we have an agreement.